Chinese/16fall_USA/Columbia_university/columbia I am coming.md

```diff
@@ -0,0 +1,5 @@
+写于2016/4/13 比利时布鲁塞尔自由大学图书馆
+经历了一个不算满意的申请季，最后决定去哥大了。
+未来两年必须努力，争取能读david blei 或者collins 的phd.
+也算正式转码了。
+[这个世界做事选对方向也很重要啊！](../the-right-way/to_be_a_coder.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.